packages/nextcloud/lib/src/helpers/news.dart

```diff
@@ -0,0 +1,16 @@
+// ignore_for_file: public_member_api_docs
+
+/// See https://github.com/nextcloud/news/blob/4a107b3d53c4fe651ac704251b99e04a53cd587f/lib/Db/ListType.php
+enum NewsListType {
+  feed(0),
+  folder(1),
+  starred(2),
+  allItems(3),
+  shared(4),
+  explore(5),
+  unread(6);
+
+  const NewsListType(this.code);
+
+  final int code;
```

Comment:
Is the code even needed?

`NewsListType.feed.index` should also yield the same (yes this requires correct ordering).